src/components/list/list.scss

```diff
@@ -143,18 +143,18 @@ md-list-item, md-list-item .md-list-item-inner {
   }
 
   & > md-icon:first-child:not(.md-avatar-icon) {
-    margin-right: $list-item-primary-width - $list-item-primary-icon-width;
+    @include rtl-prop(margin-right, margin-left, $list-item-primary-width - $list-item-primary-icon-width);
   }
   & > md-checkbox {
     width: 3 * $baseline-grid;
-    margin-left: 3px;
-    margin-right: 29px;
+    @include rtl(margin-left, 3px, 29px);
+    @include rtl(margin-right, 29px,3px);
```

Comment:
no i check and it is ok
in ltr mode is
    margin-left: 3px;
    margin-right: 29px; 
in rtl mode is
   margin-left: 29px
   margin-right: 3px